<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DiskDlg</class>
 <widget class="QDialog" name="DiskDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1366</width>
    <height>768</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>64</y>
     <width>281</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>请选择要导出的磁盘目录：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="diskDirctory_lineEdit">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>102</y>
     <width>501</width>
     <height>31</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="SourcefileWidget" name="sourcefileWidget" native="true">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>150</y>
     <width>615</width>
     <height>611</height>
    </rect>
   </property>
  </widget>
  <widget class="SourcefileWidget" name="targetfileWidget" native="true">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>150</y>
     <width>691</width>
     <height>611</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>70</y>
     <width>271</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>源目录：</string>
   </property>
  </widget>
  <widget class="QLabel" name="sourcePath_label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>110</y>
     <width>471</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>e:/</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>1170</x>
     <y>102</y>
     <width>169</width>
     <height>32</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QPushButton" name="disksSelectDirectoryBtn">
      <property name="minimumSize">
       <size>
        <width>80</width>
        <height>30</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>80</width>
        <height>30</height>
       </size>
      </property>
      <property name="text">
       <string>打开</string>
      </property>
      <property name="icon">
       <iconset resource="fastvideo.qrc">
        <normaloff>:/images/fileopen.png</normaloff>:/images/fileopen.png</iconset>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QPushButton" name="diskOutputBtn">
      <property name="minimumSize">
       <size>
        <width>80</width>
        <height>30</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>80</width>
        <height>30</height>
       </size>
      </property>
      <property name="text">
       <string>导出</string>
      </property>
      <property name="icon">
       <iconset resource="fastvideo.qrc">
        <normaloff>:/images/copyout.png</normaloff>:/images/copyout.png</iconset>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>SourcefileWidget</class>
   <extends>QWidget</extends>
   <header>sourcefilewidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="fastvideo.qrc"/>
 </resources>
 <connections/>
</ui>
